packages/vulcan-core/lib/modules/decorators/autocomplete.js

```diff
@@ -2,9 +2,9 @@ import { getCollectionByTypeName, isEmptyOrUndefined, fieldDynamicQueryTemplate,
 import get from 'lodash/get';
 
```

Comment:
Just a details but we could put decorators into "vulcan:lib", as the other stuffs related to schema are living there